src/analytics/Properties.tsx

```diff
@@ -60,6 +60,8 @@ import { RecipientType } from 'src/recipients/recipient'
 import { Field } from 'src/swap/types'
 import { Currency, CurrencyOrCREAL, StableCurrency } from 'src/utils/currencies'
 import { Awaited } from 'src/utils/typescript'
+import { AdventureCardMap } from 'src/onboarding/ChooseYourAdventure'
```

Comment:
I guess we have other screen imports here that uses Analytics, but this introduces a cyclic dependency right? 
```
ChooseYourAdventure -> ValoraAnalytics -> Properties -> ChooseYourAdventure
```
Should we avoid introducing another one?

Reply:
Yeah, I can refactor things in order to avoid this cycle.